packages/web-components/fast-components/src/tabs/tabs.ts

```diff
@@ -0,0 +1,301 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import {
+    keyCodeArrowDown,
+    keyCodeArrowLeft,
+    keyCodeArrowRight,
+    keyCodeArrowUp,
+    keyCodeEnd,
+    keyCodeHome,
+    wrapInBounds,
+} from "@microsoft/fast-web-utilities";
+
+export enum TabsOrientation {
+    vertical = "vertical",
+    horizontal = "horizontal",
+}
+
+export class Tabs extends FASTElement {
+    @attr
+    public orientation: TabsOrientation = TabsOrientation.horizontal;
+    private orientationChanged(): void {
+        this.orientation === TabsOrientation.vertical
+            ? this.classList.add("vertical")
```

Comment:
You could be able to bind this class in the template

Reply:
This would require the tab to have an `orientation` attr. I'd like to avoid that